src/common/components/entry-list-item/index.tsx

```diff
@@ -98,14 +98,16 @@ interface State {
   showMuted: boolean;
   showModMuted: boolean;
   mounted: boolean;
+  dateFormatted: string;
```

Comment:
entry vote btn component already has entry object, so maybe have this state in that component since it is related there??

Reply:
@MTSaeed So `dateFormatted` prop is not necessary anymore right?

Reply:
@MTSaeed There is couple warnings in dev console, please check screenshot. 
![image](https://user-images.githubusercontent.com/3015990/212781168-982a2dc9-3fce-4873-a236-2d30ddc4538a.png)

Reply:
@MTSaeed also there is conflict on PR